api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+<h3>Getting Started:</h3>
+<p>First of all, make sure you have all you need to get started:</p>
+<ul>
+    <li>Make sure you have your Accessy app. If you don´t, you will find all the information you need on the <a href="https://wiki.makerspace.se/Accessy">Accessy wiki page</a>.</li>
+    <li>Make sure to download Slack (from <a herf="https://play.google.com/store/apps/details?id=com.Slack&hl=en_US&pli=1">Google Play</a> or <a href="https://apps.apple.com/se/app/slack/id618783545">Apple Store</a>), since it is the main means of communication at the makerspace. If you need help, check the <a href="https://wiki.makerspace.se/Slack">Slack Wiki page</a>.</li>
+    <li>Here are links to other important online resources</li>
+    <ul>
+        <li>&#128100; <a href="https://medlem.makerspace.se/">Din medlemssida</a></li>
+        <li>&#x1F6D2; <a href="https://medlem.makerspace.se/shop">Medlemsshop</a></li>
+        <li>&#128198; <a href="https://www.makerspace.se/kalendarium">Kalendarium</a></li>
+        <li>&#128213; <a href="https://wiki.makerspace.se/V%C3%A4lkommen_till_Stockholm_Makerspace">Wiki page</a></li>
+    </ul>
+    <li>Check some of the amazing things our members have created on the <a href="https://stockholmmakerspace.slack.com/archives/C4M0MJ96K">#projekt</a> channel on Slack, to get inspired!</li>
+    <li>Join the <a href="https://stockholmmakerspace.slack.com/archives/CGQ04052Q">#events</a> channel on Slack and find out about upcoming events and courses. </li>
+</ul>
 
-<h3>I'm building something a bit bigger than a storage box, can I still store it here?</h3>
-<p>If there is space available, you can place your project on the shelves in the storage room, for a maximum of 3 months. Your project should be marked with a label for temporary storage from <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a>, otherwise it will be thrown away. We have room for purchased sheets of plywood, acrylic and the like next to the laser cutter in the large room.</p>
+<h3>Expected conduct at the space</h3>
 
-<h3>I left my project on a table over night and now it's gone!</h3>
-<p>You have to clean up after yourself. We have a cleaning service that we pay a considerable amount to mop floors and tables every week. But the cleaner does not touch tables with things on them, and does not clean floors where things are. It is very important that you clean up after yourself, so that we all can have a common, clean room. If you happened to leave things on a table, it might have been thrown away or moved to a Lost & Found box. Members help to empty the garbage. Ask a member if you don't know how to do it, or check out the <a href="https://wiki.makerspace.se/St%C3%A4drutiner">cleaning routines</a>.</p>
+<p style="text-align:center;"><i>“Be considered. Be responsible.”</i></p>
```

Comment:
```suggestion
<p style="text-align:center;"><i>“Be considerate. Take responsibility.”</i></p>
```
This is the more idiomatically correct phrasing. But best run this whole email past a native English speaker.